src/bin/restore.js

```diff
@@ -42,8 +43,19 @@ if (compilerPlugins) {
   processor.compilerPlugins.push(...requireByPath(compilerPlugins));
 }
 
+var strict = false
+if (argv.strict) {
+  strict = true;
+}
+
+function restore_with_strict() {
+  return function(...args) {
+    return processor.sourceAndRedactedToRestored(...args, strict);
+  }
+}
+
 function restore(data) {
-  return recursivelyProcessAll(processor.sourceAndRedactedToRestored.bind(processor), data);
+  return recursivelyProcessAll(restore_with_strict().bind(processor), data);
```

Comment:
(following up with https://github.com/code-dot-org/redactable-markdown/pull/63#discussion_r302211861)

Oooh, I see. In that case, how about something more explicit, like

```js
return recursivelyProcessAll((source, redacted) => processor.sourceAndRedactedToRestored(source, redacted, strict), data);
```

Reply:
Oh that is bettter!